src/main.cpp

```diff
@@ -1,8 +1,61 @@
 #include <iostream>
 #include "CourseDashboard.hpp"
+#include "Coders.hpp"
+#include <vector>
+#include <algorithm>
+
+void printEveryone(const std::vector<Coders>& tmpLibrary);
+void printUser(Coders tmpUser);
+void addUser(const Coders& tmpUser);
+void deleteUser(std::vector<Coders> tmpVec, Coders tmpUser); // WIP
+
+std::vector<Coders> codersLibrary;
```

Comment:
this vector and addUser / deleteUser and other future functions should be extracted into a new class.